waveform_collection/server.py

```diff
@@ -275,7 +275,7 @@ def gather_waveforms_bulk(lon_0, lat_0, max_radius, starttime, endtime,
                           channel, network='*', station='*', location='*',
                           time_buffer=0, merge_fill_value=0, trim_fill_value=0,
                           remove_response=False, watc_url=None,
-                          watc_username=None, watc_password=None):
+                          watc_username=None, watc_password=None, iris_only=False):
```

Comment:
As discussed, I think we should set default to iris_only=True since that is the standard FDSN server we use